### DISCUSSION TITLE
Suggestion: Consider Developing an iOS App for Easier Access

### DISCUSSION THREAD

**Discussion by melisaakilic** [dailydotdev/daily#1595] [Feature Requests]
Maybe this has been suggested before, but I couldn't find your app on iOS. I believe you should definitely have one. This platform feels like Instagram for developers, and honestly, I’d love to access it easily from my phone

**Comment by idoshamun** [comment]
Hi @melitata,

Unfortunately, iOS doesn't allow publishing our webapp to the store which is what we do with our android app.
And our team is not big enough to build and support a native app at the moment.
We'd love to have an iOS app when the time comes and we can support it
